examples/chef/common/stubs.cpp

```diff
@@ -8,6 +8,159 @@
 #ifdef EMBER_AF_PLUGIN_DOOR_LOCK_SERVER
 #include <app/clusters/door-lock-server/door-lock-server.h>
 
+class LockManager
+{
+public:
+    static constexpr uint32_t kNumEndpoints              = 1;
+    static constexpr uint32_t kNumUsersPerEndpoint       = 2;
+    static constexpr uint32_t kNumCredentialsPerEndpoint = 20;
+    static constexpr uint32_t kNumCredentialsPerUser     = 10;
+    static constexpr uint32_t kMaxNameLength             = 32;
+    static constexpr uint32_t kMaxDataLength             = 16;
+
+    struct Credential
+    {
+        bool set(DlCredentialStatus status, DlCredentialType type, chip::ByteSpan newData)
+        {
+            if (newData.size() > kMaxDataLength)
+                return false;
+            memcpy(data, newData.data(), newData.size());
+            info = EmberAfPluginDoorLockCredentialInfo{
+                status,
+                type,
+                chip::ByteSpan(data, newData.size()),
+            };
+            return true;
+        }
+
+        EmberAfPluginDoorLockCredentialInfo info = {};
```

Comment:
Shouldn't this init the info to indicate the slot is free?

Reply:
Yes, zero-init of DlCredentialStatus is kAvailable. I suppose I could make it explicit